Packs/GitLab/Playbooks/Gitlab_-_User_Permission_Changed_Alert.yml

```diff
@@ -0,0 +1,660 @@
+id: Gitlab - User Permission Changed Alert
+version: -1
+name: Gitlab - User Permission Changed Alert
```

Comment:
I think we should rename and playbook and the correlation rule to Gitlab - Guest user permission change 
to indicate a guest user in involve.